src/Shared/StackTrace/StackFrame/StackTraceHelper.cs

```diff
@@ -16,6 +17,7 @@ namespace Microsoft.Extensions.StackTrace.Sources;
 
 internal class StackTraceHelper
 {
+    [UnconditionalSuppressMessage("Trimming", "IL2026:RequiresUnreferencedCode", Justification = "StackFrame.GetMethod() is best effort to display helpful information.")]
```

Comment:
If there is only 1 method call that is the problem (like `StackFrame.GetMethod()`), you may want to refactor out the call into its own method and apply this `UnconditionalSuppressMessage` just to the specific method. The problem with suppressing 2026 for a large method is that it suppresses all warnings of that type. If some other method started warning, you wouldn't get the new warning.
This way the suppression is scoped to just the one method.